RPIPs/RPIP-10.md

```diff
@@ -81,6 +83,22 @@ Committee Governance section.
   - The MC SHALL receive feedback following those reports.
   - The MC SHOULD respond to this feedback and respect the will of the broader pDAO.
 
+### Updating this RPIP
+This is a "Living" RPIP, so that it can be kept up to date with changes to the pDAO budget or budget
+procedures. There SHALL NOT be any merged updates (beyond correcting errata and add non-normative
+clarifications) unless they are accepted by pDAO vote. The following process SHALL be followed to
+update this RPIP:
+
+1. An author SHALL make edits as desired and create a pull request; the status in the PR SHALL be
+   "Draft"
+2. An RPIP editor SHALL review it. If there are no structural issues, the status shall be changed
+   to "Review" 
+3. The vote procedure SHALL be followed as if this were an entirely new RPIP. This includes making
+   forum posts, appropriate wait times, quorums, etcetera, as detailed elsewhere.
+4. After the vote is concluded:
+   1. If the vote passed, the status SHALL be changed to "Living" and the PR SHALL be merged
+   2. If the vote failed, the status SHALL be changed to "Vote Failed" and the PR SHALL be closed 
```

Comment:
...and the branch SHOULD NOT be deleted.

Reply:
I tried an experiment and it should be fine to delete the branch. Not as easy to re-open, but the history and deltas all seem to be there.

Suggesting leaving the text as is.